libraries/node-core-library/src/test/Text.test.ts

```diff
@@ -4,26 +4,73 @@
 import { Text } from '../Text';
 
 describe('Text', () => {
-  it('Text.padEnd()', () => {
-    expect(Text.padEnd('', 5)).toEqual(      '     ');
-    expect(Text.padEnd('123', 5)).toEqual(   '123  ');
-    expect(Text.padEnd('12345', 5)).toEqual( '12345');
-    expect(Text.padEnd('123456', 5)).toEqual('123456');
+  describe('padEnd', () => {
+    it('Throws an exception if the padding character isn\'t a single character', () => {
```

Comment:
>it [](start = 4, length = 2)

"it" is deprecated. This is some old karma crap.  Jest uses "test" and "describe" is optional unless you actually have multiple sections in your test file.